### PR TITLE
Update analysis for run

### PR DESCRIPTION
**Pull request by jwarnn**
These updates seem to fix the problems with these scripts I would like to do some more testing but I wanted to open this PR in case someone else would like to try it out.

### CODE REVIEW

**Comment by jwarnn** [comment]
The scripts that I have added are working for mycosnp. Once we have a CLIA validated Granduer version running on ICA I would like to update the auto_Grandeur_ICA.py so that it work with analysis_for_run.py.

**Comment by jwarnn** [comment]
These scripts are nearly done. Just want a few new runs to go through and also need to write how to download the reads but  need to turn off daily_SARS-CoV-2_metadata.sh. Will ask for reviews soon.

**Comment by jwarnn** [comment]
Alright bugs fixed and I finally got around to updating the README. Tell me if this is going to meet the needs of the lab and if things need to be added or fixed.

**Review by tives82 — COMMENTED** [review]
Looks good!

**Review by tives82 — APPROVED** [review]

File: analysis_for_run.py
Comment (erinyoung) [inline analysis_for_run.py:167]: This is very handy

File: download_ica.py
Comment (erinyoung) [inline download_ica.py]: Can we change `/Volumes/IDGenomics_NAS/WGS_Serotyping` to `/Volumes/IDGenomics_NAS/pulsenet_and_arln/`?

File: download_reads.py
Comment (erinyoung) [inline download_reads.py]: Can we change `/Volumes/IDGenomics_NAS/WGS_Serotyping` to `/Volumes/IDGenomics_NAS/pulsenet_and_arln`?

File: download_reads.py
Comment (erinyoung) [inline download_reads.py]: I'm not seeing the Sample sheet getting downloaded. I think it's because of the space on line 73.
`--output= "%s"`  should be `--output="%s"`

File: download_reads.py
Comment (erinyoung) [reply download_reads.py]: Actually, I'd like to request the entire path to be `/Volumes/IDGenomics_NAS/pulsenet_and_arln/%s/reads`.

File: download_reads.py
Comment (erinyoung) [inline download_reads.py]: This would need to be updated when the path changes

File: MiSeq_runs_basespace.sh
Comment (erinyoung) [inline MiSeq_runs_basespace.sh]: Your python scripts make MiSeq_runs_basespace.sh unnecessary, correct? I think that means we can delete it.

**Comment by jwarnn** [comment]
I will work in those changes you suggested @erinyoung when I figure out how to fix the API calls that should be get all of the species associated with a run. Right now for some reason it is only grabbing one species.

**Comment by jwarnn** [comment]
Before this goes into production I want to add some type of logging so that we know which script failed and why. I will work on that and ask for one more review.

**Comment by jwarnn** [comment]
If anyone cares I finally figured out a logging scheme that I liked and seems to work. Hopefully I can finally get this merged and running in Prod on the linux station.

**Review by erinyoung — APPROVED** [review]
We should probably work on getting this merged so we can start building off of it.

**Comment by erinyoung** [comment]
It looks like there are merge conflicts that have to be resolved on the command line.

**Comment by jwarnn** [comment]
We have two runs currently running that will be a good test of the logging feature. I will merge this pull request if all goes well or make the need updates. Unless anyone thinks we need one more look through by another set of eyes.

**Comment by erinyoung** [comment]
I look forward to getting this merged!

**Pull request merged**
